appveyor.yml

```diff
@@ -32,4 +32,4 @@ build_script:
 
 test_script:
   - cargo test --verbose --release
-  - cargo test --verbose --release --features use-mock-crypto
+  - cargo test --verbose --release --features mock
```

Comment:
Similar to the Travis script, we can add a line like https://github.com/Fraser999/maidsafe_utilities/blob/master/appveyor.yml#L35 for the seeded rng.